src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { Box, CssBaseline } from '@material-ui/core'
+import ThemeProvider from '@material-ui/styles/ThemeProvider'
+import { createMuiTheme } from '@material-ui/core/styles'
+import { LayoutProps } from './types'
+import Header from './Header'
+
+const Layout: React.SFC<LayoutProps> = ({ children }) => {
+  return (
+    <ThemeProvider
+      theme={createMuiTheme({
+        palette: {
```

Comment:
We should move this configuration to a theme.ts file

```
export default {
  palette: {
    primary: {
      main: '#ffecb3',
      light: '#ffffe5',
      dark: '#cbba83',
      contrastText: '#000000'
    },
    secondary: {
      main: '#ffab91',
      light: '#ffddc1',
      dark: '#c97b63',
      contrastText: '#000000'
    }
  }
}

```

And then import it in this file.. `import theme from './theme'`